src/databaseInterface/src/main/java/org/vaccineimpact/api/db/Sequences.java

```diff
@@ -26,22 +26,22 @@
     /**
      * The sequence <code>public.api_access_log_id_seq</code>
      */
-    public static final Sequence<Long> API_ACCESS_LOG_ID_SEQ = new SequenceImpl<Long>("api_access_log_id_seq", Public.PUBLIC, org.jooq.impl.SQLDataType.BIGINT.nullable(false));
```

Comment:
I just re-ran the db interface generator, using db master (as there were conflicts) and all of these changes appeared - any idea why? I would have expected these to be introduced in `i1480`?